lib/pronto/formatter/checkstyle_formatter.rb

```diff
@@ -21,7 +21,7 @@ def open_xml
         @document = REXML::Document.new.tap do |d|
           d << REXML::XMLDecl.new
         end
-        @checkstyle = REXML::Element.new('checkstyle', @document)
+        @checkstyle = REXML::Element.new("checkstyle", @document)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.